src/amchart/Candle.js

```diff
@@ -13,13 +13,12 @@
         
         this._gType = "candlestick";
         
-        this._openField;
-        this._closeField;
-        this._categoryField;
-        this._lowField;
-        this._highField;
-    };
-    
+        this._openField = undefined;
+        this._closeField = undefined;
+        this._categoryField = undefined;
+        this._lowField = undefined;
+        this._highField = undefined;
```

Comment:
...no... But the previous code does nothing and I am assuming the only reason they are there is to "self document" - IMO if they are published params, then they should be removed altogether (but that should be in a separate PR)?

(Actually now that you point it out - if they are published params, then they are now just wrong  and really _need_ to be be deleted).